package.json

```diff
@@ -18,6 +18,7 @@
   "author": "Hugh Francis",
   "license": "MIT",
   "devDependencies": {
+    "ember-data": "1.13.11",
```

Comment:
Totally true! I don't know how ember-data ended up in the package.json, somehow I thought it was there before up until this point. Even asked @kellyselden why this is needed... :)